packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1079,6 +1084,13 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Theme.of(context).cardColor is used.
   final Color? color;
 
+  /// If provided, menu padding is used for the popup menu.
+  ///
+  /// If this property is null, then [PopupMenuThemeData.padding] is used.
+  /// If [PopupMenuThemeData.padding] is also null, then
+  /// EdgeInsets.symmetric(vertical: [_kMenuVerticalPadding]) is used.
```

Comment:
As this is a private variable, we should probably spell out the constant here:

```suggestion
  /// EdgeInsets.symmetric(vertical: 8.0) is used.
```